index.html

```diff
@@ -612,14 +612,14 @@ <h3>Summary of Members</h3>
         </tfoot>
       </table>
 
-      <p class="note">
-      Other {{WebAppManifest}} members
-        ({{WebAppManifest/scope}},
-        {{WebAppManifest/theme_color}},
-        {{WebAppManifest/related_applications}},
-        {{WebAppManifest/prefer_related_applications}}, and
-        {{WebAppManifest/shortcuts}}) are not currently supported by MiniApp user agents.
-      </p>
+      <div class="note" role="note" id="issue-container-generatedID"><div role="heading" class="note-title marker" id="h-note" aria-level="4"></div><p class="">
+      Other <a data-link-type="idl" data-type="dictionary" href="https://www.w3.org/TR/appmanifest/#dom-webappmanifest"><code>WebAppManifest</code></a> members
+        (<a data-link-type="idl" data-type="dict-member" href="https://www.w3.org/TR/appmanifest/#dom-webappmanifest-scope"><code>scope</code></a>,
```

Comment:
The link should probably be https://www.w3.org/TR/appmanifest/#scope-member without `data-link-type="idl" data-type="dict-member"`. Same for other members.